bin/silius/src/silius.rs

```diff
@@ -56,11 +59,18 @@ pub struct Opt {
 
     #[clap(flatten)]
     pub bundler_opts: BundlerServiceOpts,
+
+    #[clap(long)]
+    pub build_fb_signer: Option<bool>,
+
+    #[clap(long, default_value = "eth-client")]
```

Comment:
We can do that will all the values defined for this param: https://github.com/Vid201/silius/blob/main/bin/silius/src/cli.rs#L51

eth-client, flashbots